### PR TITLE
[6.5.x] RHBPMS-4141 : [ALL LANG] Unlocalized strings in 'Add or Edit New Organizational Unit' dialog.

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/RHBPMS-4141
https://issues.jboss.org/browse/RHBPMS-1165

(cherry picked from commit 783bf099ccc15044007064558c56965b744104c3)

### CODE REVIEW

File: guvnor-organizationalunit-manager/src/main/java/org/guvnor/organizationalunit/manager/client/editor/popups/EditOrganizationalUnitPopup.ui.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
     <b:Form type="HORIZONTAL">
         <b:FieldSet>
-            <b:Legend>Organizational Unit Information</b:Legend>
+            <b:Legend text='{i18n.OrganizationalUnitInformation}'></b:Legend>
             <b:FormGroup ui:field="nameGroup">
                 <b:FormLabel for="name" addStyleNames="col-md-4">
                     Name
```
Comment (manstis) [inline guvnor-organizationalunit-manager/src/main/java/org/guvnor/organizationalunit/manager/client/editor/popups/EditOrganizationalUnitPopup.ui.xml]: There are still some non-i18n'ed constants... ("Name", "Default Group ID", "Owner" in ```EditOrganizationalUnitPopup.ui.xml```)

Comment (Rikkola) [inline guvnor-organizationalunit-manager/src/main/java/org/guvnor/organizationalunit/manager/client/editor/popups/EditOrganizationalUnitPopup.ui.xml]: Well spotted, I just did a search for "text=" and "placeholder=". Fixing these.

**Comment by Rikkola** [comment]
@manstis Ok, new try.

**Comment by manstis** [comment]
+1 to merge. Thanks @Rikkola

**Comment by manstis** [comment]
(I don't want to merge here.. as it'll squash the commits.. and I believe we want to keep them separate)...

**Comment by manstis** [comment]
Merged with:

- https://github.com/droolsjbpm/guvnor/commit/1be3b2c92a64e64da89ebddc0570362811cee286
- https://github.com/droolsjbpm/guvnor/commit/559a38dc4f7ba226d37cce66d95a5dda1f9fdb01

**Comment by Rikkola** [comment]
@manstis  Thanks for taking the time for manual push. The commits look good.

**Pull request closed without merging**